mappings/net/minecraft/entity/mob/DrownedEntity.mapping

```diff
@@ -37,6 +37,15 @@ CLASS net/minecraft/class_1551 net/minecraft/entity/mob/DrownedEntity
 	FIELD field_7232 landNavigation Lnet/minecraft/class_1409;
 	FIELD field_7233 targetingUnderwater Z
 	FIELD field_7234 waterNavigation Lnet/minecraft/class_1412;
+	METHOD method_20672 isPosUnderSeaLevel (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)Z
```

Comment:
is pos really needed here? I guess it is as it doesnt use the entities pos.

Reply:
I can drop the pos part from the name.